python/spacewalk/spacewalk-backend.changes.mc.move-schema-storage

```diff
@@ -0,0 +1 @@
+- schema dir moved to /usr/share/susemanager/db
```

Comment:
Would it be possible to consider defining the path `/usr/share/susemanager/db` as a variable at the beginning of the script for better maintainability?

Reply:
no, too many different packages and languages. And the code for reading a config file is just too complicated for a 1 time usage. Additionally we do not plan to change this again. The old location was there for more than 20 years.

Reply:
ok